A4G/Donations

```diff
@@ -0,0 +1,53 @@
+---
+
+# Notes: 
+#   - Frequency: all donation entries are single donations. If you donate 
+#                monthly to a charity you will have to add new entries here on
+#                a monthly basis.
+
+- ticker: A4G
+  # ^ No POOL ID!??? Yes, the alliance would not allow duplicate tickers anyway
+  #                  and we can always pull the pool ID from the membership 
+  #                  JSONs history.
+
+  recipient: 
+    name: Save the Children
+    address: 30 Orange Street
+    tel: +44 (0)20 3272 0300
+    contact: info@savethechildren.org
+
+  ada: 2145 # <- These must be numbers. Can be decimals though. See below
+  usd: 265.3 # Only usd currency accepted, check on xe.com for exchange rate.
+  timestamp: 1599026880 
+  # ^ This is a unix timestamp of the moment you donated.
+  #   You can go to https://www.unixtimestamp.com/ to calculate your donation.
+
+  files:
+  # ^ a list of files relating to the donation. Usualy photos or screenshots 
+  #   of the receipts.
+  #   Make sure to add only relative paths that begin from the root of the 
+  #   repo (they should include your Ticker name).
+    - TICKR/2020-09-02-TGB.png
+    - TICKR/2020-09-02-photo-of-delivery.png
+
+
+  ## The following properties are optional
+
+
+  announcement_url: https://kind.software/article/spectrum-pool-spec-donations.html
+  # ^ A link to the announcement of your donation
+
+  charity_tx_id: c9a866dd665285299236164805bfdeb8e46ac20cee76407ba2bd0e2fe553b133
+  # ^ CharityReferencePaymentID etc...
+
+
+  intermediate_txs:
+  # ^ This is a list of
+    - provider: Paypal # The canonical name of the provider you used
+      tx_id: 53667941B3363534Y # The transaction ID the provider gave you 
+    - provider: Cardano
+      tx_id: 76e6d3338c2ec040b7c991d00e065d048797e2a28efc02ff27b72e51e8be3254
+    - provider: ZCash
+      tx_id: 1619b17dd4ab2e51b707a25f3befa26db593bb1cb8c4fa0102a7202760b82b06
+
+
```

Comment:
Remove or fill out these optional attributes.
Generally all comments above can be removed.